packages/common/src/transformers/BarChartDataTransformer.ts

```diff
@@ -0,0 +1,68 @@
+import { type ResultRow } from '../types/results';
+import { type BarChartConfig } from '../types/visualizations';
+import { SqlRunnerResultsTransformer } from './ResultTransformers';
+
+// TODO: Should this just be ChartDataTransformer?
+export class BarChartDataTransformer {
+    private transformer: SqlRunnerResultsTransformer;
+
+    constructor(args: { data: ResultRow[] }) {
+        this.transformer = new SqlRunnerResultsTransformer(args);
+    }
+
+    public getEchartsSpec(config: BarChartConfig) {
+        const data = this.transformer.getRows();
+        const columns = this.transformer.getColumns();
+        const { axesConfig, seriesConfig } = config;
+
+        if (config.type !== 'barChart') return {};
```

Comment:
I don't think we need this for now... let's treat this as a `barChart` until we decide what to do here